indico/modules/events/abstracts/templates/display/abstracts.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout/conference_page_base.html' %}
+{% from 'events/abstracts/management/_abstract_list.html' import render_abstract_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
```

Comment:
@DavidAndreev, you may want to have a look at this since you're working on this file